io.github.suntorytimed.camunda-modeler.appdata.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.github.suntorytimed.camunda-modeler</id>
+  <provides><id>io.github.suntorytimed.camunda-modeler.desktop</id></provides>
+  <launchable type="desktop-id">io.github.suntorytimed.camunda-modeler.desktop</launchable>
+  <name>Camunda Modeler</name>
+  <developer_name>suntorytimed</developer_name>
+  <summary>Process Modeling Tool</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>MIT</project_license>
+  <url type="homepage">https://github.com/camunda/camunda-modeler</url>
+  <description>
+    <p>Process Modeling Tool</p>
+  </description>
+  <releases>
+    <release version="5.32.0" date="2025-02-12"/> 
+  </releases>
+</component>
```

Comment:
missing mandatory screenshots, Description is too short.

Reply:
Added more details for the release